python/build_definitions/squeasel.py

```diff
@@ -22,7 +22,7 @@ class SqueaselDependency(Dependency):
     def __init__(self) -> None:
         super(SqueaselDependency, self).__init__(
             name='squeasel',
-            version='8ac777a122fccf0358cb8562e900f8e9edd9ed11-yb-1',
+            version='d83cf6d9af0e2c98c16467a6a035ae0d7ca21cb1-yb-2',
```

Comment:
Not sure why it is called d83cf6d9af0e2c98c16467a6a035ae0d7ca21cb1-yb-2. It is just the first Yugabyte-patched release off of the upstream commit d83cf6d9af0e2c98c16467a6a035ae0d7ca21cb1, so it should probably be yb-1. But can keep yb-2 for this time.